cardstack/middleware.js

```diff
@@ -103,10 +123,13 @@ class IdentityMindMiddleware {
     }
 
     let { payload } = cardstackSession;
+    let searcher = cardstackSession.userSearcher.get || cardstackSession.userSearcher;
```

Comment:
@ef4 surely this can't be right - in tests, we get `cardstackSession.userSearcher` as a function, but when using as an addon, we get an object that is shaped like `{get: () => {} }` instead.